README.md

```diff
@@ -34,6 +34,24 @@ lein run bundle.tar.gz --input '{"user": {"roles": ["admin"]}}'
 Note that the above constitutes a simple flow for development and testing only. Production artifacts, or a fixed API for
 integrations, will be here at a later point in time.
 
+### Java
+
+### Evaluating an entrypint in a plan file
+
+```java
+var file = new File("path/to/my/plan.json");
+var input = Map.of("user", "alice");
+Map<String, ?> data = Map.of();
+var allowed = Jarl.builder(file)
+        .build()
+        .getPlan("my_policy/allow")
```

Comment:
Not related to this change, but this makes me think we should use `entrypoint` rather than `getPlan` as that would be more consistent with our docs, and IMHO a better name. WDYT?

Reply:
Perhaps 🤔 . In general, the naming is all over the place, I think: the IR serialization is referred to a a plan, but it actually contains multiple plans. We need to strike a good balance with how this is described in OPA. I don't see a big problem with saying you lookup plans by their entrypoint as it makes sense to me to say you execute/evaluate a `plan`, as opposed to an `entrypoint`. The problem for me is what to call the plan container, as `IR` isn't descriptive to anyone othar than a select few; here it's just called `Jarl`.

Reply:
When building the plan(s), you'll use:

* `--target plan` (singular)
* `--entrypoint` 

So whether this is "true" or not, I think it makes sense to stick to that convention set by OPA.